src/winetricks

```diff
@@ -9439,6 +9439,33 @@ verify_dotnet48()
 
 #----------------------------------------------------------------
 
+w_metadata dotnetcore2 dlls \
+    title="MS .NET Core Runtime 2.1 LTS" \
+    publisher="Microsoft" \
+    year="2020" \
+    media="download" \
+    file1="dotnet-runtime-2.1.17-win-x86.exe" \
+    installed_file1="$W_PROGRAMS_WIN/dotnet/dotnet.exe"
+
+load_dotnetcore2()
+{
+    # Official version, see https://dotnet.microsoft.com/download/dotnet-core/2.1
+    w_download https://download.visualstudio.microsoft.com/download/pr/73a0ea97-57d6-4263-ac36-ea3a9b373bcf/cd5f7e08e749c1d3468cdae89e4518de/dotnet-runtime-2.1.17-win-x86.exe 706a7f0ad998c3c1b321e89e4bcd6304bef31c95c83eda119e8d4adccccbc915
+    unattended_args="/quiet"
+
+    w_try_cd "$W_CACHE"/"$W_PACKAGE"
+    w_try "$WINE" "$file1" ${unattended_args}
+
+    if [ "$W_ARCH" = "win64" ]; then
+      # Also install the 64-bit version
+      file1="dotnet-runtime-2.1.17-win-x64.exe"
```

Comment:
There's no need to reset the variable here.

Side note: if you got this idea from some documentation, please let me know where, I'd like to update it).

Reply:
So I may have gone a bit rogue here.  `shellcheck` started to throw errors when I tried to call the 64-bit installer `file2` in the metadata, but stopped complaining when I moved the declaration into the if block.  I then noticed that there where no other `$file2` expansions anywhere in the code, so I decided to just reset the variable I already had.

I have noticed that not many verbs use the `$file1` expansion. Should I change both of these to the executable name and avoid this issue entirely?

Reply:
It's used by winetricks_is_cached(), which is used to show verbs already have their files downloaded. It only supports one file (currently, though I don't really have plans to expand it, I don't think anyone really uses that feature).

I would say keep file1 in the metadata, and you can (optionally) use it for the 32-bit install. In the 64-bit block, don't set a variable, just do:
```
      w_download ...
      w_try "$WINE" executable.exe ...
```

Reply:
Actually, don't worry about it, I've got some changes that I'm about to push that will conflict, so I'll fix this up for you and push.

Thanks for working on this!